internal/nodes/dequeue_ingestion.go

```diff
@@ -154,6 +154,11 @@ func DequeueIngestion(key string, fullsync bool) {
 		svcNames, svcFound := schema.GetParentServices(name, namespace, key)
 		if svcFound && utils.CheckIfNamespaceAccepted(namespace) {
 			for _, svcNSNameKey := range svcNames {
+				svcNSName := strings.Split(svcNSNameKey, "/")
```

Comment:
Please refer to my reply to your other review above. 
All VSes associated with invalid LB services are automatically deleted and all VSes associated with valid LB services are kept.

Since k8s does not allow updation of `spec.loadBalancerClass` field, either user will have to remake the service to define this field explicitly, or set `defaultLBController` to true in values.yaml to make sure all svcs without this field are ingested